Region  Number of points Non psr (%) psr (%) Lon min Lon max Lat min Lat max
 South          41216400      98.79%   1.21%   0.009 359.991 -89.994 -55.188
 North          41216400      98.92%   1.08%   0.009 359.991  55.188  89.994
 Total          82432800      98.86%   1.14%   0.009 359.991 -89.994  89.994

Plot saved to ../../data/plots
Found 12 CSV files in ../../data/CSVs/combined

Merged DF:
    Latitude   Longitude  psr  ...        M3    Elevation  Label
0 -55.187865  135.000000    0  ...  0.943844 -4649.972669      2
1 -55.192961  134.991073    0  ...  0.942655 -4649.964036      2
2 -55.198056  134.982143    0  ...  0.941465 -4649.955403      3
3 -55.203150  134.973210    0  ...  0.940276 -4649.946770      3
4 -55.208243  134.964274    0  ...  0.939087 -4649.938136      2

[5 rows x 9 columns]

Merged df cols: Index(['Latitude', 'Longitude', 'psr', 'Diviner', 'MiniRF', 'LOLA', 'M3',
       'Elevation', 'Label'],
      dtype='object')
percentage of psr value of 1 in merged df: 1.14%
Proportion of each label with psr value of 1:
Traceback (most recent call last):
  File "/rds/general/user/as5023/home/.local/lib/python3.11/site-packages/pandas/core/indexes/base.py", line 3805, in get_loc
    return self._engine.get_loc(casted_key)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "index.pyx", line 167, in pandas._libs.index.IndexEngine.get_loc
  File "index.pyx", line 196, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 7081, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 7089, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'label'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/rds/general/user/as5023/home/Lunar_point_model/data_processing/jobscripts/../psr_test.py", line 178, in <module>
    main(args)
  File "/rds/general/user/as5023/home/Lunar_point_model/data_processing/jobscripts/../psr_test.py", line 150, in main
    num_psrs = np.sum((df_merged['psr'] == 1) & (df_merged['label'] == label))
                                                 ~~~~~~~~~^^^^^^^^^
  File "/rds/general/user/as5023/home/.local/lib/python3.11/site-packages/pandas/core/frame.py", line 4102, in __getitem__
    indexer = self.columns.get_loc(key)
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rds/general/user/as5023/home/.local/lib/python3.11/site-packages/pandas/core/indexes/base.py", line 3812, in get_loc
    raise KeyError(key) from err
KeyError: 'label'

Job finished on Sun  8 Dec 12:56:11 GMT 2024
Total runtime: 181 minutes and 10 seconds

====================================
CPU Time used: 02:58:33
CPU Percent: 99%
Memory usage: 92829128kb
Approx Power usage: 0.0356
Walltime usage: 03:01:21

====================================
